build.sbt

```diff
@@ -58,20 +65,29 @@ mappings.in(Universal) ++= resourceDirectory
     val src = resourceDir / "docs"
     val dest = "/docs"
 
-    val docFiles = (for {
-      path <- better.files.File(src.toPath).listRecursively()
-      if !path.isDirectory
-    } yield path.toJava -> path.toString.replaceFirst(src.toString, dest)).toSeq
+    def listRecursively(f: File): Array[File] = {
+      val these = f.listFiles
```

Comment:
use a java `FileWalker`

Reply:
something like:
```
Files.walk(Paths.get(path))
        .filter(Files::isRegularFile)
```
should make the work.

Reply:
I reimplemented it using java.nio.Files

Reply:
:+1: